examples/plotting/line_on_off.py

```diff
@@ -10,7 +19,7 @@
 from bokeh.plotting import figure, show
 
 p = figure()
-props = dict(line_width=4, line_alpha=0.7)
+props = {"line_width":4, "line_alpha":0.7}
```

Comment:
I'm -0 on this change (I'd be happy to just revert it) but if we do keep it, then it needs to match stylistically with the rest of the codebase:
```suggestion
props = {"line_width": 4, "line_alpha": 0.7}
```

Reply:
All changes executed. I matched the style of the codebase by adding the space.